src/flamenco/runtime/tests/fd_vm_test.c

```diff
@@ -254,15 +254,15 @@ do{
   vm->reg[8]  = input->vm_ctx.r8;
   vm->reg[9]  = input->vm_ctx.r9;
   // vm->reg[10]  = input->vm_ctx.r10; // set in fd_vm_init
-  vm->reg[11] = input->vm_ctx.r11;
+  // vm->reg[11]  = input->vm_ctx.r11;  // set in fd_vm_init
```

Comment:
Q: this change is related to the dynamic stack frames, right? Should this be feature/sbpf version gated?

Reply:
ah, I see 👍